### DISCUSSION TITLE
Is it possible to apply a predefined zoom?

### DISCUSSION THREAD

**Discussion by lelukas** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#553] [Q&A] [answered]
There's map_locked, but is there a way to set a default zoom value for the map besides locking it or not?

**Comment by PiotrMachowski** [comment, marked as answer]
Yes, it's possible using [cropping](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card#cropping-options) in Map Card or (preferably) using [trimming](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#map-transformation-configuration) in Map Extractor.
